enos/ansible/roles/bench/tasks/rally.yml

```diff
@@ -38,21 +38,32 @@
     command: rally deployment create --fromenv --name=discovery
 
 - name: Copy rally scenarios
-  copy: src={{ bench.location }} dest=/root/rally_home/ owner=655500
+  copy: src={{ bench.scenario_location }} dest=/root/rally_home/ owner=655500
+
+- name: Create plugin directory
+  when: bench.plugin_location is defined
```

Comment:
Maybe put the when at the end of the task to be consistent with the following tasks